batch/batch/front_end/front_end.py

```diff
@@ -1109,15 +1204,17 @@ async def create_batch_fast(request, userdata):
     batch_and_bunch = await request.json()
     batch_spec = batch_and_bunch['batch']
     bunch = batch_and_bunch['bunch']
+
     batch_id = await _create_batch(batch_spec, userdata, db)
-    try:
-        await _create_jobs(userdata, bunch, batch_id, app)
-    except web.HTTPBadRequest as e:
-        if f'batch {batch_id} is not open' == e.reason:
-            return web.json_response({'id': batch_id})
-        raise
-    await _close_batch(app, batch_id, user, db)
-    return web.json_response({'id': batch_id})
+
+    update_id = batch_spec.get('update_id') or secret_alnum_string(8)
```

Comment:
These four operations should probably use the same transaction. This is probably the same for `update-fast` as well.